code/modules/admin/admin_verbs.dm

```diff
@@ -1228,3 +1229,34 @@ GLOBAL_LIST_INIT(view_runtimes_verbs, list(
 	vis.set_content(ui_dat.Join(""))
 	vis.open(FALSE)
 
+
+/client/proc/create_rnd_restore_disk()
+	set name = "Create RnD Backup Restore Disk"
+	set category = "Event" // Im putting this in event because the name is long and will offset everything
+
+	if(!check_rights(R_ADMIN))
+		return
+
+	var/list/targets = list()
+
+	for(var/rnc_uid in SSresearch.backups)
+		var/datum/rnd_backup/B = SSresearch.backups[rnc_uid]
+
+		targets["[B.last_name] - [B.last_timestamp]"] = rnc_uid
+
+	var/choice = input(src, "Select a backup to restore", "RnD Backup Restore") as null|anything in targets
```

Comment:
Same thing here as above, would it be okay to switch these to tgui?

Reply:
Not for the admin verb